lib/foreman_tasks/widget_manager.rb

```diff
@@ -0,0 +1,16 @@
+module ForemanTasks
+  class WidgetManager
+    DEFAULT_WIDGETS = [
+      {:template=>'foreman_tasks/tasks/dashboard/tasks_status', :sizex=>5, :sizey=>1, :name=> N_('Tasks Status table')},
```

Comment:
visually, :sizex => 6 worked more for me (so that I have the two-column view) in the dashboard)